exercises/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,11 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return arr.shift(); // complete this statement
 }
 
 function last(arr) {
-  return; // complete this statement
+  return arr.pop(); // complete this statement
 }
```

Comment:
Amanda, I think you intended by using shift and pop to remove the first and last elements. I might be misunderstood the question but to get the values you can apply index numbers in return as
    return arr[0]; 
I made it like that to get the answers 1,3 and Joe.

Reply:
Yeah there are different methods of getting the answer, I just needed a bit of practice with pop() and shift()

Reply:
plz. don't modify an array in a function unless it's explicitly requested, it's acceptable here, but the function names don't indicate the function call will change the array for example